src/example-types/filters/tagsQuery.js

```diff
@@ -80,6 +80,29 @@ let filter = ({ tags, join, field, exact }) => ({
   },
 })
 
+let result = async (node, search) => {
+  let { tags, field } = node
+
+  let aggs = {
+    aggs: {
+      tags: {
+        filters: {
+          filters: F.arrayToObject(
+            _.get('word'),
+            tag => filter({ ...node, tags: [tag] }),
+            tags
+          ),
+        },
+      },
+    },
+  }
```

Comment:
I would recommend factoring this out as a `buildQuery` method like we've done in some of the other types so we can also write tests asserting that we're building the elastic DSL we expect

Reply:
gotcha -- wil do

Reply:
updated